### ISSUE TITLE
miners dont do their job

### ISSUE THREAD

**Issue by hunnybunny21** [ldtteam/minecolonies#770]
minecolonies-universal-1.11-0.6.3282

miners dont do their job
and havent announced if they need anything

**Comment by Zoidey** [comment]
What do you mean exactly? Are they not doing anything when you hire them or after they have dug out the mine shaft?

Make sure they have a Pick, Shovel, Torches, Oak Fences and Oak Planks, Oak Slabs and Ladder. So they can Build the mine shaft and/ or the tunnels.

Also make sure your sure your Citizen Hut (House) is high enough level for you miner or they won't level up (I made this mistake). Not sure if any of that is much help from you as I'm no expert.

**Comment by hotluck** [comment]
I think I have the same problem. In my case it is my second miner that is not working. He started but not for long. He needed a ladder so I gave him some (which are in his inventory) but just stays in the citizens hut since then. I tried recalling him but he just stays by the chest. I looked in the log and could only see 2 messages to do with the miner which are straight after I gave him the ladders. Would this be the cause of the problem?

> [19:41:43] [Client thread/INFO]: [CHAT] Miner at Hotluck City: Joshua C. Courtenay: I need 1 Ladder!
[19:51:12] [Client thread/INFO]: [com.mawnt.storageboats.event.StorageEventHandler:onPlayerRightClick:54]: EntityCitizen['Joshua C. Courtenay'/202, l='MpServer', x=53.02, y=65.00, z=183.72]
[19:51:12] [Server thread/INFO]: [com.mawnt.storageboats.event.StorageEventHandler:onPlayerRightClick:54]: EntityCitizen['Joshua C. Courtenay'/202, l='Colony World', x=53.02, y=65.00, z=183.72]

Hope this helps :)

BTW, I'm using 1.10.2-0.6.3317 version.

**Comment by Raycoms** [comment]
He has cobblestone in his inventory as well?

2017-04-07 12:07 GMT-03:00 Amanda <notifications@github.com>:

**Comment by hunnybunny21** [comment]
giving cobble stone made them start working
i think the oak wood planks stairs and fence may help too

for some reason the miners were not announcing what they need

**Comment by hunnybunny21** [comment]
on my second miner
it seems like all they were waiting for was cobblestone
they already had ladders torches pick axe

**Comment by ziocho** [comment]
I've this issue as well today on 1.11-0.6.3360! Given miner everything they've stated before: pickaxe, torches, planks, slabs, ladders, fence. Miner's inventory and quarry inventory are both clear of anything else. Restarted game, recalled miner during a clear day, fired and rehired her. She'd either idle in the citizen's hut, or idle at her mining hut/townhall if I recalled her. It was only until I gave the miner some cobblestone that they began working again, maybe for plugging lava and off she went! Unfortunately, there isn't a chat prompt that shows up, or if it does it isn't recurring.
